cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,54 @@
-describe('', () => {
-  before(() => {
+const faker = require('faker');
+let article; 
+let user;
 
+describe('Article flow suit', () => {
+  beforeEach(() => {
+    article = {
```

Comment:
You generate data for creating an article in the config file using generateUser, so it's better to assign the generated object to a variable to avoid code duplication

Reply:
done.